cart-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,14 @@
+server:
+  port: 9096
+spring:
+  application:
+  name: CartService
+  datasource:
+    url: jdbc:mysql://${MYSQL_HOST:localhost}:3306/mydb
```

Comment:
database name should be meaningful. You can take `cart_db`

Reply:
Done